utm_tracker/models.py

```diff
@@ -3,38 +3,29 @@
 from typing import Type
 
 from django.conf import settings
-from django.contrib.auth import get_user_model
 from django.db import models
 from django.db.models.base import Model
-from django.http import HttpRequest
 from django.utils import timezone
 
-User = get_user_model()
+from .types import UtmParamsDict
 
 
 class LeadSourceManager(models.Manager):
-    def create_from_request(
-        self, user: Type[Model], request: HttpRequest
+    def create_from_utm_params(
```

Comment:
This will break platform usage in one place. No big deal.

Reply:
I did do it with the platform code in the other window, and I _think_ it only means replacing this code,

```diff
-    persist_lead_sources_for_user(user=user, request=request)
+    flush_utm_params(user, request.session)
```

and removing the `persist_lead_sources_for_user` function.